src/components/common/ListBox.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import S from './ListBox.style';
+import NavigateNoticeTextButton from './NavigateNoticeTextButton';
+import NavigateTextButton from './NavigateTextButton';
+type ListBoxItemProps = {
+  label: string;
+  value: string;
+  onPress?: () => void;
+};
+
+type ListBoxProps = {
+  items: ListBoxItemProps[];
+};
```

Comment:
사소하지만 `ListBoxItemProps`분리안하는 게 난 좋을 것 같아요 ListBoxItemProps 가 재사용되지 않고 ListBox 컴포넌트를 사용하는 쪽에서 item의 타입을 ide에서 바로 확인가능할 것 같아요.
```suggestion
type ListBoxItemProps = {
  label: string;
  value: string;
  onPress?: () => void;
};

type ListBoxProps = {
  items: {
    label: string;
    value: string;
    onPress?: () => void;
  }[];
};
```

Reply:
props 관련 부분 반영했고,
<img width="301" alt="image" src="https://github.com/user-attachments/assets/90690621-2587-4b04-aa3a-a58fc9131fc2">

value 또한 옵셔널하게 받도록 변경해서 위와 같이 조건으로 적용했습니다.

label, value, onPress 있을 때,
label, value만 있을 때,
label만 있을 때

세가지 적용했습니다. 감사합니다.